Makefile.in

```diff
@@ -243,6 +243,7 @@ SUBDIRS = \
 # default targets for the pure Fortran version
 DEFAULT = \
 	xdecompose_mesh \
+	xdecompose_mesh_paral \
```

Comment:
Done.

On 02/10/2016 07:24 PM, Elliott Sales de Andrade wrote:
> In Makefile.in
> <https://github.com/geodynamics/specfem3d/pull/694#discussion_r52500644>:
>
>> @@ -243,6 +243,7 @@ SUBDIRS = \
>>  # default targets for the pure Fortran version
>>  DEFAULT = \
>>  	xdecompose_mesh \
>> +	xdecompose_mesh_paral \
>
> Why not name it |xdecompose_mesh_mpi| like the file name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/694/files#r52500644>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr